base/Dockerfile.ubi8

```diff
@@ -67,6 +67,11 @@ RUN microdnf install yum \
 ADD --chown=appuser:appuser target/${ARTIFACT_ID}-${PROJECT_VERSION}-package/share/doc/* /usr/share/doc/${ARTIFACT_ID}/
 ADD --chown=appuser:appuser target/${ARTIFACT_ID}-${PROJECT_VERSION}-package/share/java/${ARTIFACT_ID}/* /usr/share/java/${ARTIFACT_ID}/
 
+# Operator dependencies for metrics integration
+RUN mkdir -p /opt/javaagents
+ADD --chown=appuser:appuser target/${ARTIFACT_ID}-${PROJECT_VERSION}-package/share/java/${ARTIFACT_ID}/jolokia-jvm-*-agent.jar /opt/javaagents/.
+ADD --chown=appuser:appuser target/${ARTIFACT_ID}-${PROJECT_VERSION}-package/share/java/${ARTIFACT_ID}/jmx_prometheus_javaagent-*.jar /opt/javaagents/.
```

Comment:
These are placed in `/opt/caas/lib` [here](https://github.com/confluentinc/confluent-platform-images/blob/master/components/common/Dockerfile#L29-L37). But I placed them in `/opt/javaagents` here since that seemed to be more intuitive.

@xnulinu Please let me know if we should place them in `/opt/caas/lib` or anywhere else.

Reply:
@jhlee8 `/opt/javaagents` seems better than what we used to have before. Thanks